pkgs/tools/misc/sheldon/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchCrate
+, rustPlatform
+, pkg-config
+, openssl
+, installShellFiles
+, Security
```

Comment:
`error: Function called without required argument "Security" at ...`

fix in https://github.com/NixOS/nixpkgs/pull/200123